test/common/common/mem_block_builder_test.cc

```diff
@@ -71,25 +71,24 @@ TEST(MemBlockBuilderTest, AppendUint32) {
 TEST(MemBlockBuilderTest, AppendOneTooMuch) {
   MemBlockBuilder<uint8_t> mem_block(1);
   mem_block.appendOne(1);
-  EXPECT_DEATH({ mem_block.appendOne(2); },
 #ifdef ENVOY_CONFIG_COVERAGE
-               "" // For some reason, this test under coverage generates a list of testdata/*.
+  // For some reason, this test under coverage generates a list of testdata/*.
+  EXPECT_DEATH({ mem_block.appendOne(2); }, "");
 #else
-               ".*insufficient capacity.*"
+  EXPECT_DEATH({ mem_block.appendOne(2); }, ".*insufficient capacity.*");
```

Comment:
Thanks for doing this and sorry about this (this was mine). One minor suggestion:

```
#ifdef  ENVOY_CONFIG_COVERAGE
  static const char expected_death_regex[] = "";
#else 
  static const char expected_death_regex[] = ".*insufficient capacity.*";
#endif 
  EXPECT_DEATH({ mem_block.appendOne(2); }, expected_death_regex);
```
This seems better from a don't-repeat-yourself perspective. Here & below.

Reply:
Resolved as suggested, thanks!